anger.js

```diff
@@ -26,15 +26,17 @@ function anger (opts) {
     ? identifier
     : (payload) => get(payload, identifier)
 
-  for (var i = 0; i < clients.length; i++) {
+  for (let i = 0; i < clients.length; i++) {
     clients[i] = new Client(opts.url)
+    clients[i]['_angerIndex'] = i
```

Comment:
Wasn't 100% sure I could start the variable with `_`, but thinking on it now, ofc I can when I use lodash and underscore. _facepalm_